### ISSUE TITLE
The Beaver's Spawn Weight Is Too High

### ISSUE THREAD

**Issue by whiteheket** [bodhiahn/eager-beavers#2]
I think the spawn weight of this mob is too high, the fishes' spawn weight in vanilla minecraft's river is only 25, and other mobs' are no more than 20. Recently i was playing a mcpack contains your mod and nearly 40 beavers spawn near my house in one place. My fps drop from 120 to 40. please add config to it or change the spawn weight. Thank you very mach ! :D

**Comment by AnimatorXX** [comment]
Yes. That mrpack is my modpack [AnimteCraft: Lankeren](https://modrinth.com/modpack/animatecraft-lankeren). Besides, they are a bit too destructive, all wooden blocks near the rivers are ruined lol. We love your mod, it deserves to be famous, plz fix the bugs!

**Comment by bodhiahn** [comment]
Hey guys! just figured out that I can see these comments sorry for the late response!! The newest update should fix all these problems, both with spawning and them eating everything!
